src/main/java/com/sogeti/SoSenseiPsyche/model/Game.java

```diff
@@ -1,42 +1,75 @@
 package com.sogeti.SoSenseiPsyche.model;
 
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Scanner;
 
 public class Game {
     public static final int CODE_LENGTH = 4;
+    public static final int MAX_ATTEMPTS = 12;
+    public int attempt = 0;
+    private Code secretCode;
+    private String[] newGuess;
+    private List<String> guesses;
 
 // for testing purpose
 
     public void startGame() {
-        System.out.println("Welcome to Mastermind!");
-        System.out.println("Available colors: ");
+        guesses = new ArrayList<>();
+        secretCode = new Code();
+        System.out.println("Welcome to So Sensei Psyche!");
+        System.out.println("You have 12 attempts to crack the code.");
+        System.out.println("You can choose from the following colors: " + Arrays.toString(Color.values()).replace("[", "").replace("]", ""));
```

Comment:
It would be handy to repeat this message so that the player doesn't have to scroll too much

Reply:
i agree. It should be printed on every attempt